src/main/java/seedu/mindmymoney/command/UpdateCommand.java

```diff
@@ -121,7 +121,11 @@ public void updateExpenditure() throws MindMyMoneyException {
             }
             LocalDate date = LocalDate.parse(inputTime, DateTimeFormatter.ofPattern("dd/MM/yyyy"));
             String newTime = date.format(DateTimeFormatter.ofPattern("dd/MM/yyyy"));
-
+            if (isSimilarExpenditure(indexToUpdate, newPaymentMethod, newCategory, newDescription, newAmountAsFloat,
+                    newTime)) {
+                throw new MindMyMoneyException("Expense fields to be updated is similar to the expense in the list.\n"
+                        + "Please make sure the field descriptions you want to change are different.");
+            }
             //Create new expenditure object to substitute in
```

Comment:
If I recall correctly, comments must have a whitespace after //
```suggestion
            // Create new expenditure object to substitute in
```